modules/nw-egress-router-configmap.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift_sdn/configuring-egress-router-configmap.adoc
+
+[id="configuring-egress-router-configmap_{context}"]
+= Configuring an egress router destination mappings with a ConfigMap
+
+For a large or frequently-changing set of destination mappings, you can use a config map to externally maintain the list.
+An advantage of this approach is that permission to edit the config map can be delegated to users without `cluster-admin` privileges. Because the egress router pod requires a privileged container, it is not possible for users without `cluster-admin` privileges to edit the pod definition directly.
```

Comment:
There's a return here - meant to be two paragraphs?

Reply:
@jeana-redhat, I cheat; Sometimes it's easier to do sentence-per-line. Our guidelines don't explicitly forbid it, I don't think.